0_portfolio_input_check_functions.R

```diff
@@ -869,7 +868,7 @@ get_and_clean_fin_data <- function(fund_data){
   
   # Financial Data
   fin_data_raw <- read_file(paste0(analysis_inputs_path,"/security_financial_data.rda")) %>% as_tibble()
-  
```

Comment:
I see a bunch of changes like this, which likely come from my configuring RStudio to remove trailing white space. This seems to be a kind of best practice but I'm happy to drop it in the interest of avoiding this "false positives". But before I do that, I'll run a styler in a separate PR.

![image](https://user-images.githubusercontent.com/5856545/95361902-ef6bd100-0892-11eb-89bc-7a3e562a6a96.png)

Reply:
I can set that as well, it seems reasonable